dash_core_components/CircleSpinner.py

```diff
@@ -0,0 +1,58 @@
+# AUTO GENERATED FILE - DO NOT EDIT
+
+from dash.development.base_component import Component, _explicitize_args
```

Comment:
The Loading component uses the various spinner flavours so it makes sense to have a component for each flavour, but I'm not sure exposing the various flavours as top-level dcc components makes sense.